templates/Makefile

```diff
@@ -169,6 +168,9 @@ install: ansible_install scripts_install component_install $(ENMASSE_BUNDLE) $(M
 package: prepare install
 	tar -czf build/$(INSTALLNAME).tgz -C build $(INSTALLNAME)
 
+deploy:
+	mvn deploy:deploy-file -Dfile=build/$(INSTALLNAME).tgz
```

Comment:
What is the idea of deploying this file to the remote repository?  Might this cause problems in some build environments?

Reply:
The idea is that it contains the build 'zip'. I'm not sure if we're actually using it, so I can remove it.